src/main/java/uk/gov/companieshouse/ocr/api/statistics/StatisticsDTO.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.ocr.api.statistics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class StatisticsDTO {
```

Comment:
Should be StatisticsDto not StatisticsDTO

Reply:
Updated